src/Orchard.Web/Modules/Orchard.Projections/Migrations.cs

```diff
@@ -282,5 +287,21 @@ public class Migrations : DataMigrationImpl {
 
             return 4;
         }
+
+        public int UpdateFrom4() {            
+            SchemaBuilder.AlterTable("PropertyRecord", table => table
+                .AddColumn<string>("RewriteAsEmptyRule", c => c.Unlimited())
+            );
+            var queries = _contentManager.Query<QueryPart, QueryPartRecord>().List();
```

Comment:
Can you remove this _migration_ code, if the new column is null then it should be handled in the rest of the code.